TSOClient/tso.client/UI/Panels/UILotControl.cs

```diff
@@ -322,6 +328,19 @@ void vm_OnDialog(FSO.SimAntics.Model.VMDialogInfo info)
                     options.Buttons = new UIAlertButton[] { new UIAlertButton(UIAlertButtonType.OK, b0Event, info.Yes), new UIAlertButton(UIAlertButtonType.Cancel, b1Event, info.Cancel) };
                     options.GenericAddition = new UIColorPicker();
                     break;
+                case VMDialogType.FSOJob:
+                    VMAvatar avatar = (VMAvatar)info.Caller;
+
+                    //Find their current active job
+                    VMTSOAvatarState state = (VMTSOAvatarState)avatar.TSOState;
+                    KeyValuePair<short, VMTSOJobInfo> jobInfo = state.JobInfo.FirstOrDefault(x => x.Value.StatusFlags == 1);
+                    if (jobInfo.Key != 0)
+                    {
+                        JobInformation JobInfo = new JobInformation((int)jobInfo.Value.Level, (int)jobInfo.Key, (int)jobInfo.Value.Experience); //Job Grade, Job Type, Experience
+                        var jobInfoAlert = new UIJobInfo(JobInfo);
+                        jobInfoAlert.Show();
+                    }
```

Comment:
This is not tied into the rest of the dialog system. It's fine if the dialog does not block (if you haven't encountered any issues, it probably doesn't) but in future please try to make your dialog modifications a GenericAddition so that the blocking dialog stuff still works.

Reply:
I will keep this in mind for any future changes and might refactor as I become a bit more comfortable in the future, I have no had any blocking issues during testing.